tutorials/tts/Fastpitch_Training_GermanTTS.ipynb

```diff
@@ -0,0 +1,1112 @@
+    "    phoneme_record = records[i].copy()\n",
+    "    phoneme_record[\"normalized_text\"] = phonemized[i]\n",
+    "    phoneme_record[\"phoneme\"] = 1\n",
+    "    new_records.append(phoneme_record)\n",
+    "    \n",
+    "with open(output_manifest_filepath, \"w\") as f:\n",
+    "    for r in new_records:\n",
+    "        f.write(json.dumps(r) + '\\n')\n",
+    "```\n",
+    "\n",
+    "To better understand the phonemize method, refer to the docs [here](https://github.com/bootphon/phonemizer/blob/master/phonemizer/backend/base.py#L137).\n",
+    "\n",
+    "Run the above script for train, test and val records, resulting in `train_phonemes_manifest.json`, `test_phonemes_manifest.json` and `val_phonemes_manifest.json` respectively.\n",
+    "\n",
+    "We are effectively doublng the size of our dataset. Each original record maps on to two records, one with original `normalized_text` field value and `phonemes` set to 0 and another with phonemized text and `phonemes` flag set to 1.\n",
+    "\n",
+    "Example of input record:\n",
+    "```json\n",
+    "{\"audio_filepath\": \"/Data/thorsten-de/wavs/e50eb02c25353f85549900d2fc1e0e32.wav\", \"duration\": 2.409977, \"text\": \"Geht die Schandtat auf sein Konto?\", \"normalized_text\": \"Geht die Schandtat auf sein Konto?\"}\n",
+    "```\n",
+    "And corresponding output records:\n",
+    "```json\n",
+    "{\"audio_filepath\": \"/Data/thorsten-de/wavs/e50eb02c25353f85549900d2fc1e0e32.wav\", \"duration\": 2.409977, \"text\": \"Geht die Schandtat auf sein Konto?\", \"normalized_text\": \"Geht die Schandtat auf sein Konto?\", \"phoneme\": 0}\n",
+    "{\"audio_filepath\": \"/Data/thorsten-de/wavs/e50eb02c25353f85549900d2fc1e0e32.wav\", \"duration\": 2.409977, \"text\": \"Geht die Schandtat auf sein Konto?\", \"normalized_text\": \"\\u0261e\\u02d0t di\\u02d0 \\u0283ant\\u0251\\u02d0t a\\u028af za\\u026an k\\u0254nto\\u02d0 \", \"phoneme\": 1}\n",
+    "```\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a7245360",
+   "metadata": {},
+   "source": [
+    "## 4. Create dataset config\n",
+    "\n",
+    "Most of the configuration remains the same as described in [FastPitch_MixerTTS_Training.ipynb](FastPitch_MixerTTS_Training.ipynb) except:\n",
+    "1. The `text_tokenizer._target_` is set to `nemo.collections.tts.torch.tts_tokenizers.GermanCharsTokenizer` class, and pass `phonemes: true`, which will extend the `de_alphabet` to include IPA symbols, resulting in `abcdefghijklmnopqrstuvwxyzäöüßʊʃŋɜːɛɾəɪçɔøɡœɑÜ„1Q̃ɒʒÄɹÖʌθàó̈ðéɐá`.\n",
+    "\n",
+    "2. The `text_normalizer.lang` is set to `de`\n",
+    "\n",
+    "Final config looks like:\n",
+    "\n",
+    "```yaml\n",
+    "name: \"ds_for_fastpitch_align\"\n",
+    "\n",
+    "manifest_filepath: \"train_manifest.json\"\n",
+    "sup_data_path: \"sup_data\"\n",
+    "sup_data_types: [ \"align_prior_matrix\", \"pitch\" ]\n",
+    "whitelist_path: \"nemo_text_processing/text_normalization/de/data/whitelist.tsv\"\n",
```

Comment:
May want to point out that the whitelist path needs to be set to the German one as well.